### PR TITLE
freshvm: Use qcow2 img to improve startup time

### PR DESCRIPTION
**Pull request by toabctl**
Instead of copying and converting from qcow2 to raw, just
use a backing file with qcow.
Also remove some dead code.

### CODE REVIEW

File: scripts/jenkins/ci1/freshvm [hunk 1]
```diff
@@ -20,16 +20,13 @@ if ! [ -f $p/$flavor.img -o -f $p/$flavor.qcow2 ]; then
...
 
 virsh destroy $n
-if [ "$n" = "cleanvm" ] ; then
-    dd if=/dev/zero of=/dev/vg0/$n-data count=16
-    if [ -e $p/$flavor.qcow2 ] ; then
-        time qemu-img convert $p/$flavor.qcow2 /dev/vg0/$n
-    else
-        time dd if=$p/$flavor.img of=/dev/vg0/$n bs=64k
-    fi
+dd if=/dev/zero of=/dev/vg0/$n-data count=16
```
Comment (aplanas) [inline scripts/jenkins/ci1/freshvm]: I found `fallocate` a lot faster than `dd`. In this case maybe is not important, but if there are other `dd` we can think to replace them

**Comment by bmwiedemann** [comment]
:+1: but, I would be interested in how much the runtime of the remaining test changes

File: scripts/jenkins/ci1/freshvm (hunk 1 above)
Comment (jdsn) [inline scripts/jenkins/ci1/freshvm]: and not neccessary all VMs deployed this way have a *-data volume, so I recommend to add:
`if [[ -e /dev/vg0/$n-data ]] ; then ...`

Comment (toabctl) [inline scripts/jenkins/ci1/freshvm]: > and not neccessary all VMs deployed this way have a *-data volume, so I recommend to add:
> if [[ -e /dev/vg0/$n-data ]] ; then ...

Done in https://github.com/toabctl/automation/commit/793830145eb2111517d4299e8036db8dca73ee19

**Comment by jdsn** [comment]
:+1:

**Pull request merged**